contracts/empty/README.md

```diff
@@ -0,0 +1,4 @@
+# Empty Contract
+
+This is a minimal contract to test against. It has no functionality and only has
+the minimum required to be a valid contract.
```

Comment:
Can you add a compilation command to `contracts/README.md` and `packages/vm/README.md` to make the build reproducible and smaller?

Reply:
I added the commands for the new contract and also made some adjustments for the floaty contract that were still missing.